spec/migrations/20190226184255_check_guid_uniqueness_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_migration
+
+describe CheckGuidUniqueness do
+  let(:automate_workspace) { migration_stub(:AutomateWorkspace) }
+  let(:miq_ae_workspace) { migration_stub(:MiqAeWorkspace) }
+  let(:miq_policy) { migration_stub(:MiqPolicy) }
+  let(:miq_region) { migration_stub(:MiqRegion) }
+  let(:miq_widget) { migration_stub(:MiqWidget) }
+  let(:service) { migration_stub(:Service) }
+  let(:service_template) { migration_stub(:ServiceTemplate) }
+
+  migration_context :up do
+    %i(automate_workspace miq_ae_workspace miq_policy miq_region miq_widget service service_template).each do |obj|
+      it "#{obj} dup resets guid" do
+        guid = SecureRandom.uuid
+        klass = send(obj)
+        obj1 = klass.create!(:guid => guid)
+        obj2 = klass.new(:guid => guid)
```

Comment:
Can you also create a third object with a unique guid to ensure it isn't updated?